.travis.yml

```diff
@@ -1,3 +1,38 @@
-language: node_js
-node_js:
-  - '0.12'
+language: generic
```

Comment:
Since you are specifying a language for each option in the matrix there is no need for this here.